src/common/models/IAzureClient.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+"use strict";
+import { IPartialList } from "./IPartialList";
+
+export interface IAzureClient<T> {
```

Comment:
is there similar interface defined in azure-arm-containerregistry/lib/models?

Reply:
Good point. Actually these snippets are based VSCode Azure Account SDK's sample code which is service-agnostic. I will update it to be more specific to our scenarios and thus more concise. Thanks.